### ISSUE TITLE
Perhaps convert to the "awesome" format?

### ISSUE THREAD

**Issue by sgbaird** [blaiszik/Materials-Databases#4]
- https://github.com/kjappelbaum/awesome-chemistry-datasets
- https://github.com/tilde-lab/awesome-materials-informatics

Instructions at https://github.com/sindresorhus/awesome

**Comment by blaiszik** [comment]
Yes, this is a legacy project, that I haven't maintained for years. But, perhaps we should make an awesome list of these resources!

**Comment by sgbaird** [comment]
I just noticed the last commit is 8 years ago. Surprising how many platforms were out there nearly a decade ago! If either of us get the opportunity to write a review that has a focus or section on materials databases, that might be a good time to convert and update this, and provide a link to it in the paper as an ongoing list (especially with community contributions).

I'm planning to do the same with the low-cost SDL manuscript.
